db/migrate/20170512233333_drop_miq_server_rhn_mirror.rb

```diff
@@ -24,5 +24,19 @@ def up
         change.save!
       end
     end
+
+    if Rails.env.production? && File.exist?('/var/www/miq/vmdb')
+      say_with_time("Removing files created by RHN Mirror role") do
+        require 'fileutils'
+        require 'linux_admin'
+
+        LinuxAdmin::FSTab.instance.entries.delete_if { |e| e.mount_point == "/repo" }
```

Comment:
Can you `require 'linux_admin'` somewhere to show it's usage?  I'm thinking ahead to when this is in the separate repo.